tests/NHSD.BuyingCatalogue.API.IntegrationTests/Steps/Entities/SolutionEntitySteps.cs

```diff
@@ -110,7 +110,7 @@ public static async Task LastUpdatedHasUpdatedOnMarketingContact(string solution
         }
 
 
-        [Given(@"Solution have following details")]
+        [Given(@"solutions have the following details")]
         public static async Task GivenSolutionDetailExist(Table table)
```

Comment:
Let's rename this method so that it matches the text, e.g. `GivenSolutionsHaveTheFollowingDetails`.

Reply:
Method has been renamed to `GivenSolutionsHaveTheFollowingDetails`